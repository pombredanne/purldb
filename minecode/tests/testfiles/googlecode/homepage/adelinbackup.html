

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"domainName": null, "projectName": "adelinbackup", "profileUrl": null, "projectHomeUrl": "/p/adelinbackup", "loggedInUserEmail": null, "token": null, "relativeBaseUrl": "", "assetHostPath": "http://www.gstatic.com/codesite/ph", "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 adelinbackup -
 
 
 QT4 GUI for boxbackup - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fadelinbackup%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fadelinbackup%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/adelinbackup">
 <a href="/p/adelinbackup/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/adelinbackup/"><span itemprop="name">adelinbackup</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/adelinbackup/"><span itemprop="description">QT4 GUI for boxbackup</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/adelinbackup/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/adelinbackup/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/adelinbackup/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/adelinbackup/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/adelinbackup/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/adelinbackup/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/adelinbackup/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/adelinbackup/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.gnu.org/licenses/old-licenses/gpl-2.0.html" rel="nofollow">GNU GPL v2</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:GUI">GUI</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:backup">backup</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/107863326616779067217/">nes...@gmail.com</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">1 committer</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_filedownload-g16 goog-inline-block vt"></div>
 </div>
 <span><b>Downloads</b></span>
 </div>
 <ul class="pslist">
 
 <li class="psmeta nowrap">
 <a title="Download" href="//adelinbackup.googlecode.com/files/abgui-0.2.7.tar.gz" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//adelinbackup.googlecode.com/files/abgui-0.2.7.tar.gz', 'Type-Source, Featured']);">
 </a>
 <a href="downloads/detail?name=abgui-0.2.7.tar.gz"
 title="Adelin Backup GUI 0.2.7">abgui-0.2.7.tar.gz</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//adelinbackup.googlecode.com/files/ablibrary-0.0.1.tar.gz" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//adelinbackup.googlecode.com/files/ablibrary-0.0.1.tar.gz', 'Type-Source, Featured']);">
 </a>
 <a href="downloads/detail?name=ablibrary-0.0.1.tar.gz"
 title="Adelin Backup Library 0.0.1">ablibrary-0.0.1.tar.gz</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//adelinbackup.googlecode.com/files/abtray-0.0.1.tar.gz" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//adelinbackup.googlecode.com/files/abtray-0.0.1.tar.gz', 'Type-Source, Featured']);">
 </a>
 <a href="downloads/detail?name=abtray-0.0.1.tar.gz"
 title="Adelin Backup Tray Agent 0.0.1">abtray-0.0.1.tar.gz</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//adelinbackup.googlecode.com/files/abtraynotify-0.0.1.tar.gz" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//adelinbackup.googlecode.com/files/abtraynotify-0.0.1.tar.gz', 'Featured, Type-Source']);">
 </a>
 <a href="downloads/detail?name=abtraynotify-0.0.1.tar.gz"
 title="Adelin Backup Tray Notifier 0.0.1">abtraynotify-0.0.1.tar.gz</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//adelinbackup.googlecode.com/files/abwizard-0.0.1.tar.gz" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//adelinbackup.googlecode.com/files/abwizard-0.0.1.tar.gz', 'Featured, Type-Source']);">
 </a>
 <a href="downloads/detail?name=abwizard-0.0.1.tar.gz"
 title="Adelin Backup Wizard 0.0.1">abwizard-0.0.1.tar.gz</a>
 </li>
 
 <li class="psmeta"><a href="downloads/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p>AdelinBackup is a BoxBackup GUI written in C++ QT4. It contains boxbackup code with BSD license. Source code at <a href="http://github.com/nesaro/adelin/" rel="nofollow">http://github.com/nesaro/adelin/</a> </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

