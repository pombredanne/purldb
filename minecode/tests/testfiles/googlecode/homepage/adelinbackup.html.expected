{'content_license': None,
 'content_license_url': None,
 'description': u'QT4 GUI for boxbackup',
 'downloads_url': u'http://code.google.com/p/adelinbackup/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/adelinbackup/feeds',
 'home_url': u'http://code.google.com/p/adelinbackup/',
 'id': u'adelinbackup',
 'issues_url': u'http://code.google.com/p/adelinbackup/issues/list',
 'keywords': [],
 'labels': [u'GUI', u'backup'],
 'license': u'GNU GPL v2',
 'license_url': u'http://www.gnu.org/licenses/old-licenses/gpl-2.0.html',
 'name': u'adelinbackup',
 'peoples_url': u'http://code.google.com/p/adelinbackup/people/list',
 'vcs_url': u'http://code.google.com/p/adelinbackup/source/checkout',
 'wiki_url': u'http://code.google.com/p/adelinbackup/w/list'}